NDcPP_v2_2e.adoc

```diff
@@ -2085,25 +2151,20 @@ To ensure audit requirements are properly met, a DTLS receiver may need to monit
 
 If the TOE acts as the server during the claimed DTLS sessions, the ST author should claim the corresponding FCS_DTLSS_EXT requirements. In this case the TOE needs to claim at least the FCS_DTLSS_EXT.1 requirements in chap. B.3.1.1 (no support for mutual authentication). If the TOE acts as DTLS server and in addition also supports mutual authentication, the FCS_DTLSS_EXT.2 requirements in chap. A.7.1.1 need to be claimed in addition. If the TOE acts as both a client and server during the claimed DTLS sessions, the ST author should claim the corresponding FCS_DTLSC_EXT and FCS_DTLSS_EXT requirements.
 
-*FCS_DTLSC_EXT.1 DTLS Client Protocol Without Mutual Authentication*
+*FCS_DTLSC_EXT.1 DTLS Client Protocol*
 
-*FCS_DTLSC_EXT.1.1* The TSF shall implement [selection: _DTLS 1.2 (RFC 6347), DTLS 1.0 (RFC 4347)_] supporting the following ciphersuites:
+*FCS_DTLSC_EXT.1.1* The TSF shall implement [selection: _DTLS 1.3 (RFC tbd), DTLS 1.2 (RFC 6347)] and [selection: DTLS 1.0 (RFC 4347), no additional DTLS version_] supporting the following ciphersuites:
 
 * [selection:
-** [.underline]#_select supported ciphersuites from List 1] and no other ciphersuites_.#
-
-].
+** [.underline]#_select supported ciphersuites for DTLS 1.0 and 1.2 from List 1]_#
+** [.underline]#_select supported ciphersuites for DTLS 1.3 from List 2] and no other ciphersuites_.#
```

Comment:
@plughy2 Add a hard line break before '] and no other ciphersuites'.

Reply:
Added a line break